src/documentation/examples/Dropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import Vue from "vue";
+import Component from "vue-class-component";
+
+import { Dropdown } from "fsxa-ui";
+import { Option } from "@/types/components";
+
+const testOptions: Option[] = [
+  { key: "opt1", label: "Option_1", path: "path1" },
+  { key: "opt2", label: "Option_2", path: "path2" },
+];
+
+const handleClick = (item: Option) => {
+  console.log(item.label);
+};
+
+@Component
+export default class App extends Vue {
+  currentItem: any = null;
```

Comment:
Either remove the local currentItem state or use it in your example.

Reply:
Removed.